includes/class-wc-calypso-bridge-woocommerce-admin-features.php

```diff
@@ -97,6 +98,53 @@ public function filter_woocommerce_body_classes( $classes ) {
 		return $classes;
 	}
 
+	/**
+	 * Add is-woocommerce-home body class when url changes between wc-admin pages.
+	 *
+	 * @since   1.9.9
+	 *
+	 * @return void
+	 */
+	public function filter_woocommerce_body_classes_js() {
+		?>
+		<script>
+			document.addEventListener( 'DOMContentLoaded', function() {
+
+				// Bail out earlier.
+				if ( ! document.body.classList.contains( 'woocommerce_page_wc-admin' ) ) {
+					return;
+				}
+
+				let url = location.href;
+				document.body.addEventListener( 'click', ( event ) => {
+
+					// Bail out early.
+					if ( 'A' !== event.target.tagName ) {
```

Comment:
What about `button`s? Perhaps omitting this change would work better for us. We can't be sure what triggered the page change in the DOM.

Reply:
My initial thought was to limit this script to the wp-admin context. Thus searching for the `#root` element in the DOM is enough. 

Hint: We might not even need the DOM Ready if the `#root` element exists at first paint; I would also test this perf.

Reply:
Indeed, great observation about the buttons

> My initial thought was to limit this script to the wp-admin context. Thus searching for the #root element in the DOM is enough.

Checking for the existence of `woocommerce_page_wc-admin` works as expected, and there is no need for the `#root` element